commands.js

```diff
@@ -896,6 +901,10 @@ var commands = exports.commands = {
 	},
 	lockhelp: ["/lock OR /l [username], [reason] - Locks the user from talking in all chats. Requires: % @ & ~"],
 
+	forcelock: function () {
+		CommandParser.commands.lock.apply(this, arguments);
```

Comment:
Or you could just use the same alias system we use for other commands and put `forcelock: 'lock',`